pkgs/tools/package-management/nix-eval-jobs/default.nix

```diff
@@ -8,6 +8,7 @@
   curl,
   nix,
   nlohmann_json,
+  libarchive,
```

Comment:
I need to see why this fails without this, it feels unexpected to me

Reply:
I would think we can strip out propogated deps just as we do in the nix-eval-jobs upstream build.